Makefile

```
@@ -0,0 +1,191 @@
+			jsonlint "$$file" >/dev/null || { \
+				echo "JSON syntax error in $$file"; exit 1; }; \
+		elif command -v jq >/dev/null 2>&1; then \
+			jq empty "$$file" >/dev/null || { \
+				echo "JSON syntax error in $$file"; exit 1; }; \
+		else \
+			echo "No JSON linter found, skipping $$file"; fi' \
+		sh {} \;
+	@echo "JSON files passed linting!"
+
+# Check EditorConfig compliance
+lint-editorconfig:
```

checkmake (0.3.2)

[warning] Target body for "lint-editorconfig" exceeds allowed length of 5 lines (11).

(maxbodylength)